justfile

```diff
@@ -0,0 +1,26 @@
+# These are a few scripts for your convenience. To run them, use Just:
+# https://github.com/casey/just
+
+# -*- mode: make-gmake-mode; -*-
+
+debug:
+    go install --race
+
+release:
+    go install
+
+fix-dgo:
+    #!/usr/bin/env sh
+    go get github.com/bwmarrin/discordgo
+    cd "$GOPATH/src/github.com/bwmarrin/discordgo"
+    git checkout develop
+    go install
+
+cross-compile:
+    #!/usr/bin/env sh
+
+    # This requires a script only I, Mnpn, have.
+    # This will not work for you.
+    # pls ignore.
+    cd "$GOPATH/bin"
+    ./Cross\ Compile.sh DiscordConsole discordconsole-team
```

Comment:
Why is this not checked in?

Reply:
I don't have access to the script anymore, but I suppose @Mnpn03 could check it in (although IIRC it largely assumes a bunch of structure)

Reply:
Can someone please remind me about this later, thanks!

Reply:
@Mnpn03 Do you know what happened with the guy who invented the door-knocker?

Reply:
who’s there?

Reply:
He won the :no_bell: price.

Reply:
:bell: Anyway, this is your friendly reminder to commit the script

Reply:
Thanks, I've added it!